scripts/init.d-attach-static-vdis

```diff
@@ -16,6 +16,8 @@ STATE_DIR=/etc/xensource/static-vdis
 # Needed for interface-reconfigure below
 . /etc/xensource-inventory
 
+RETVAL=0
```

Comment:
How do you ensure that this value is not overwritten once it has been set? I think it would be cleaner to let functions return a value rather than using a global variable. You also might want to use https://www.shellcheck.net/.